lib/user_agent/browsers/webkit.rb

```diff
@@ -2,12 +2,12 @@ class UserAgent
   module Browsers
     class Webkit < Base
       def self.extend?(agent)
-        agent.detect { |useragent| useragent.product == 'AppleWebKit' }
+        agent.detect { |useragent| useragent.product =~ /\AAppleWebKit\z/i }
       end
 
       def browser
         if os =~ /Android/
-          'Android'
+          'Browser'
```

Comment:
I was on the fence about changing it to be 'Browser' to be honest, Android is easy enough to understand.